hardhat.config.ts

```diff
@@ -14,7 +14,9 @@ dotenv.config();
 
 const config: HardhatUserConfig = {
   etherscan: {
-    apiKey: 'HHHQV1FM9HVSK82JMPEEBG44PR24CM3B5U',
+    apiKey: {
+      bscTestnet: 'HHHQV1FM9HVSK82JMPEEBG44PR24CM3B5U',
```

Comment:
maybe it's better to put the api key into `.env.example`?

Reply:
agree, also etherscan allows to give multiple keys based on env, why not create a place holder for key on the BSC Mainnet?

Reply:
is a relatively sensitive information, so it is placed in the local env file